packages/crayons-core/src/components/email-select/email-select.tsx

```diff
@@ -0,0 +1,740 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Listen,
+  Method,
+  Prop,
+  State,
+  h,
+  Watch,
+  Fragment,
+} from '@stencil/core';
+import { range, uniq } from 'lodash';
```

Comment:
Hi, can u use `lodash/es` instead of just `lodash` . Just make sure tree shaking works here.